eng/targets/Helix.Common.props

```diff
@@ -11,17 +11,18 @@
         <HelixAvailblePlatform Include="OSX" />
         <HelixAvailblePlatform Include="Linux" />
 
-        <HelixAvailibleTargetQueue Include="Windows.10.Amd64.ClientRS4.VS2017.Open" Platform="Windows" />
-        <HelixAvailibleTargetQueue Include="Windows.10.Amd64.EnterpriseRS3.ASPNET.Open" Platform="Windows" EnableByDefault="false" />
-        <HelixAvailibleTargetQueue Include="OSX.1012.Amd64.Open" Platform="OSX" />
-        <HelixAvailibleTargetQueue Include="Ubuntu.1810.Amd64.Open" Platform="Linux" />
-        <HelixAvailibleTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" />
-        <HelixAvailibleTargetQueue Include="Centos.7.Amd64.Open" Platform="Linux" />
-        <HelixAvailibleTargetQueue Include="Debian.8.Amd64.Open" Platform="Linux" />
-        <HelixAvailibleTargetQueue Include="Debian.9.Amd64.Open" Platform="Linux" />
-        <HelixAvailibleTargetQueue Include="Redhat.7.Amd64.Open" Platform="Linux" />
-        <HelixAvailibleTargetQueue Include="Fedora.28.Amd64.Open" Platform="Linux" />
-
-        <!--  TODO: re-enable Debian.9.Arm64.Open and Ubuntu.1804.Arm64.Open    -->
+        <HelixAvailibleTargetQueue Include="Windows.10.Amd64.ClientRS4.VS2017.Open" Platform="Windows" Arch="x64" />
+        <HelixAvailibleTargetQueue Include="Windows.10.Arm64.Open" Platform="Windows"  Arch="arm" />
+        <HelixAvailibleTargetQueue Include="Windows.10.Amd64.EnterpriseRS3.ASPNET.Open" Platform="Windows"  Arch="x64" EnableByDefault="false" />
+        <HelixAvailibleTargetQueue Include="OSX.1012.Amd64.Open" Platform="OSX" Arch="x64" />
+        <HelixAvailibleTargetQueue Include="Ubuntu.1810.Amd64.Open" Platform="Linux" Arch="x64" />
+        <HelixAvailibleTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" Arch="x64" />
+        <HelixAvailibleTargetQueue Include="Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-0a0ebdd-20190312215438" Platform="Linux" Arch="arm" />
+        <HelixAvailibleTargetQueue Include="Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-0a0ebdd-20190312220351" Platform="Linux" Arch="arm" />
```

Comment:
Can we also add a Alpine ARM64 Queue? https://github.com/dotnet/corefx/blob/56115730ea260dd27cf74818dff858cb8424e61a/eng/pipelines/linux.yml#L122

I just added a linux-musl-arm64 configuration for aspnetcore.